cypress/tests/ui/bankaccounts.spec.ts

```diff
@@ -87,9 +98,11 @@ describe("Bank Accounts", function () {
 
     cy.visit("/bankaccounts");
     cy.wait("@getBankAccounts");
+    cy.percySnapshot();
```

Comment:
I am not sure why this snapshot is needed, since you have a few assertions and another snapshot after this line